ibis/pandas/api.py

```diff
@@ -9,17 +9,23 @@ def connect(dictionary):
     return PandasClient(dictionary)
 
 
-def from_dataframe(df):
+def from_dataframe(df, name='df', client=None):
     """
     convenience function to construct an ibis table
     from a DataFrame
```

Comment:
Can you mark this as experimental? I'm not sure we'll want to keep this particular version around indefinitely. Let's see what shakes out after a few releases with people using this.